packages/solid-query/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": "tannerlinsley",
+  "dependencies": {
+    "@tanstack/query-core": "4.3.9"
+  },
+  "description": "Hooks for managing, caching and syncing asynchronous and remote data in Solid",
+  "devDependencies": {
+    "@types/jscodeshift": "^0.11.3",
+    "jscodeshift": "^0.13.1"
+  },
+  "exports": {
+    ".": {
+      "browser": "./build/lib/index.mjs",
+      "default": "./build/lib/index.js",
+      "import": "./build/lib/index.mjs",
+      "node": "./build/lib/index.js",
+      "require": "./build/lib/index.js",
+      "solid": "./build/solid/index.js",
+      "types": "./build/lib/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "build/lib/*",
+    "build/umd/*",
+    "build/solid/*",
+    "src",
+    "codemods",
+    "!codemods/jest.config.js",
+    "!codemods/**/__testfixtures__",
+    "!codemods/**/__tests__"
```

Comment:
is this just copy-paste, because I don't think we have codemods yet for solid-query 😅

Reply:
yes this was copy paste. I started by copying the entire react-query folder so this should be removed.

Reply:
I have removed all the codemod bits so we should be good here too